examples/find_and_avoid/controllers/supervisor/supervisor.py

```diff
@@ -1,12 +1,14 @@
 import math
 
+import gym
 import numpy as np
-
-import utilities as utils
 from deepbots.supervisor.controllers.supervisor_emitter_receiver import \
     SupervisorCSV
 from deepbots.supervisor.wrappers.keyboard_printer import KeyboardPrinter
-from deepbots.supervisor.wrappers.tensorboard_wrapper import TensorboardLogger
+from deepbots.supervisor.wrappers.tensorboard_wrapper import \
+    TorchTensorboardLogger
```

Comment:
Is this <code>TensorboardLogger</code>?
I cannot run the code with <code>TorchTensorboardLogger</code>.
[Link to deepbots/supervisor/wrappers/tensorboard_wrapper](https://github.com/aidudezzz/deepbots/blob/851cba96c71ea13736dbe161ec90c230b193356d/deepbots/supervisor/wrappers/tensorboard_wrapper.py#L7)

Reply:
Tensorboard logger has to be updated accordingly. So I remove this for now